src/index.ts

```diff
@@ -315,3 +328,18 @@ export class ProjectConfig {
   }
 
 }
+
+// Gets the json schema for polymer.json, generated from the typescript
+// interface for runtime validation. See the build script in package.json for
+// more info.
+const getSchema: () => jsonschema.Schema = (() => {
```

Comment:
This way is more obviously correct. Yeah we're only hiding it from ourselves, but this way you don't have to think about the bug of accessing a potentially uninitialized variable.